### PR TITLE
Delete unused CI scripts

### PR DESCRIPTION
**Pull request by dnfield**
Needs https://flutter-review.googlesource.com/c/recipes/+/15820 for the iOS side

None of these are used anymore - the firebase_xctest was never used and only would have worked from a local workstation. However, it wouldn't work any more with the way Scenario app is built now. The main blocker to testing iOS on firebase is signing - if we want to test on a real device on iOS we have to figure that out first and this script would have to be re-written anyway.

Part of https://github.com/flutter/flutter/issues/84618

### CODE REVIEW

**Review by blasten — APPROVED** [review]
Nice!

**Review by jmagman — APPROVED** [review]
LGTM once the recipes land and the "Scenario App Integration Tests" tests pass.

**Comment by dnfield** [comment]
Need to update scenarios.py too - that's in https://flutter-review.googlesource.com/c/recipes/+/15821

**Pull request merged**
